tests/unit/test_cli_unit.py

```
@@ -0,0 +1,111 @@
+    result = runner.invoke(cli.app, [])
+    assert result.exit_code == 0
+    assert "Usage" in result.stdout
+
+
+def test_transcribe_basic(monkeypatch: pytest.MonkeyPatch, tmp_path: Path) -> None:
+    """
+    Verify that transcribe resolves input paths, delegates to the transcribe implementation, and returns the returned output paths.
+    """
+    audio = tmp_path / "a.wav"
+    audio.write_text("x")
+    monkeypatch.setattr(cli, "RESOLVE_INPUT_PATHS", lambda files: [audio])
```

Ruff (0.14.8)

Unused lambda argument: `files`

(ARG005)